src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -10501,84 +10492,89 @@ static ColliderQuadInit D_808546A0 = {
+    static u8 sSwordItemIds[] = { ITEM_SWORD_MASTER, ITEM_SWORD_KOKIRI };
+    s32 swordItem = sSwordItemIds[(void)0, gSaveContext.save.linkAge];
+    s32 swordItemAction = sItemActions[swordItem];
 
     Player_DestroyHookshot(this);
     Player_DetachHeldActor(play, this);
 
-    this->heldItemId = item;
-    this->nextModelGroup = Player_ActionToModelGroup(this, itemAction);
+    this->heldItemId = swordItem;
+    this->nextModelGroup = Player_ActionToModelGroup(this, swordItemAction);
 
-    Player_InitItemAction(play, this, itemAction);
+    Player_InitItemAction(play, this, swordItemAction);
     func_80834644(play, this);
 
-    if (arg2 != 0) {
+    if (playSfx) {
         Player_PlaySfx(this, NA_SE_IT_SWORD_PICKOUT);
     }
 }
 
-static Vec3f D_808546F4 = { -1.0f, 69.0f, 20.0f };
+void Player_StartMode_TimeTravel(PlayState* play, Player* this) {
+    static Vec3f sPedestalPos = { -1.0f, 69.0f, 20.0f };
 
-void Player_StartMode1(PlayState* play, Player* this) {
     Player_SetupAction(play, this, Player_Action_8084E9AC, 0);
     this->stateFlags1 |= PLAYER_STATE1_29;
-    Math_Vec3f_Copy(&this->actor.world.pos, &D_808546F4);
+
+    Math_Vec3f_Copy(&this->actor.world.pos, &sPedestalPos);
     this->yaw = this->actor.shape.rot.y = -0x8000;
+
     LinkAnimation_Change(play, &this->skelAnime, this->ageProperties->unk_A0, 2.0f / 3.0f, 0.0f, 0.0f, ANIMMODE_ONCE,
                          0.0f);
     Player_StartAnimMovement(play, this,
                              PLAYER_ANIM_MOVEMENT_RESET_BY_AGE | ANIM_FLAG_UPDATE_XZ | ANIM_FLAG_UPDATE_Y |
                                  ANIM_FLAG_DISABLE_CHILD_ROOT_ADJUSTMENT | ANIM_FLAG_ENABLE_MOVEMENT |
                                  ANIM_FLAG_OVERRIDE_MOVEMENT);
+
     if (LINK_IS_ADULT) {
-        func_80846720(play, this, 0);
+        Player_TakeOutSword(play, this, false);
     }
+
     this->av2.actionVar2 = 20;
 }
 
-void Player_StartMode3(PlayState* play, Player* this) {
+void Player_StartMode_Door(PlayState* play, Player* this) {
     Player_SetupAction(play, this, Player_Action_8084F9A0, 0);
     Player_StartAnimMovement(play, this,
                              ANIM_FLAG_UPDATE_XZ | ANIM_FLAG_UPDATE_Y | ANIM_FLAG_ENABLE_MOVEMENT |
                                  ANIM_FLAG_ADJUST_STARTING_POS | ANIM_FLAG_OVERRIDE_MOVEMENT);
 }
 
-void Player_StartMode4(PlayState* play, Player* this) {
+void Player_StartMode_Grotto(PlayState* play, Player* this) {
     func_808389E8(this, &gPlayerAnim_link_normal_jump, 12.0f, play);
     Player_SetupAction(play, this, Player_Action_8084F9C0, 0);
     this->stateFlags1 |= PLAYER_STATE1_29;
     this->fallStartHeight = this->actor.world.pos.y;
     OnePointCutscene_Init(play, 5110, 40, &this->actor, CAM_ID_MAIN);
 }
 
-void Player_StartMode7(PlayState* play, Player* this) {
+void Player_StartMode_KnockedOver(PlayState* play, Player* this) {
     func_80837C0C(play, this, PLAYER_HIT_RESPONSE_KNOCKBACK_LARGE, 2.0f, 2.0f, this->actor.shape.rot.y + 0x8000, 0);
```

Comment:
Outside the scope of this PR, but I wonder if `PLAYER_HIT_RESPONSE_KNOCKBACK_LARGE` could have `KNOCKED_OVER` in its name somehow for consistency here.

Even more out of scope: In MM, there's an extra knockback that's "smaller" than "KNOCKBACK_SMALL" where the upper body flinches, but the lower body/movement is unaffected, i.e. no small hop occurs. So small/large doesn't work too well over there

Reply:
if movement isn't affected, then it wouldn't be a knockback would it?

Reply:
would be a "flinch"?

Reply:
I think knockback is a common term in the context of damage (which is how these were named, cause thats generally how its used)

Using a damage knockback for this use is kind of a trick. The area fade blocks the beginning of it, and by the time you see link, it looks like he already started in the air and got thrown onto the floor. Which is what they are going for with this start mode, since its used when you get caught by guards of some kind and get thrown out of a place.

All that to say I think "knocked over" describes its specific use case as a start mode well, while "knockback" desceibes its more general use as a damage response well

Reply:
Yeah, that makes sense for this function.

I'll leave the MM knockback discussion for the MM channels but the extra "flinch" type is part of `PlayerKnockbackType`, so knockback would inherently be in the name if that word is used.